# HG changeset patch
# User Clemens Buchacher <drizzd@aon.at>
# Date 1369937542 14400
#      Thu May 30 14:12:22 2013 -0400
# Node ID 122d3f62e179ba044d47e58510905719220d8706
# Parent  e38a0fa08368d4dd6f3a0dfd20cbe28cf8555f73
do not include C++ header in extern "C" context (bug #38746)

* mex.h: Include mexproto.h outside of extern "C" block.

diff -r e38a0fa08368 -r 122d3f62e179 src/mex.h
--- a/src/mex.h	Mon May 27 21:41:57 2013 +0200
+++ b/src/mex.h	Thu May 30 14:12:22 2013 -0400
@@ -64,6 +64,8 @@
 
 #define mxMAXNAME 64
 
+#include "mexproto.h"
+
 #if defined (__cplusplus)
 extern "C" {
 #endif
@@ -74,8 +76,6 @@
 void mexFunction (int nlhs, mxArray* plhs[], int nrhs, const mxArray *prhs[]);
 #endif
 
-#include "mexproto.h"
-
 /* V4 floating point routines renamed in V5.  */
 #define mexIsNaN mxIsNaN
 #define mexIsFinite mxIsFinite
